src/Development/IDE/Types/Options.hs

```diff
@@ -94,6 +94,9 @@ data IdeOptions = IdeOptions
     --   Otherwise, return the result of parsing without Opt_Haddock, so
     --   that the parsed module contains the result of Opt_KeepRawTokenStream,
     --   which might be necessary for hlint.
+  , optCustomDynFlags :: DynFlags -> DynFlags
```

Comment:
Is this for hls and other downstream users of ghcide?

Reply:
yep